11

christienetto opened the pull request:
Implemented new query Expense_participants, which uses a new schema expense_participants and updates the GroupsIDGet route's items response to include the participants in items' group division.

christienetto committed:
[server+web] Add group overview route #4

christienetto committed:
[server] expense participants server adjustments #11

Kuchteq reviewed (commented):
Please rebase with the main branch

christienetto committed:
[Server] Expense participants in GroupsIDItemsPost #38

When posting Items, the items schema has a participants memeber struct
list which is also uploaded
